models/magazines.py

```diff
@@ -5,7 +5,7 @@
 class Magazine(LibraryItem):
     __tablename__ = 'magazines'
     id = db.Column(db.ForeignKey('library_item.id'), primary_key=True)
-    year = db.Column(db.Integer)
+    year = db.Column(db.String(128))
```

Comment:
Let's change that to Date type.

Reply:
Magazines sheet in our xlsx file contains text values in the Year column. I didn't want to interfere with the original file, but If you consider this as desirable I am ready to adjust these values.

Reply:
We need that for filtering and ordering. Only Integer and Date give us proper tools for doing that. String is imho the worst option. Integer will take less storage, but date seems more natural in that context.